angle_min: 15
cam_id: 0
control_rate: 100

D: [0.0, 0.0, 0.0, 0.0, 0.0]

object_pts: [[6.825897, 6.760612, 4.402142],
              [1.330353, 7.122144, 6.903745],
              [-1.330353, 7.122144, 6.903745],
              [-6.825897, 6.760612, 4.402142],
              [5.311432, 5.485328, 3.987654],
              [1.789930, 5.393625, 4.413414],
              [-1.789930, 5.393625, 4.413414],
              [-5.311432, 5.485328, 3.987654],
              [2.005628, 1.409845, 6.165652],
              [-2.005628, 1.409845, 6.165652],
              [2.774015, -2.080775, 5.048531],
              [-2.774015, -2.080775, 5.048531],
              [0.000000, -3.116408, 6.097667],
              [0.000000, -7.415691, 4.070434]]
reprojectsrc: [[10.0, 10.0, 10.0],
                [10.0, 10.0, -10.0],
                [10.0, -10.0, -10.0],
                [10.0, -10.0, 10.0],
                [-10.0, 10.0, 10.0],
                [-10.0, 10.0, -10.0],
                [-10.0, -10.0, -10.0],
                [-10.0, -10.0, 10.0]]
line_pairs: [[0, 1], [1, 2], [2, 3], [3, 0],
              [4, 5], [5, 6], [6, 7], [7, 4],
              [0, 4], [1, 5], [2, 6], [3, 7]]
